user_tools/docs/legacy-user-tools-dataproc.md

```diff
@@ -0,0 +1,777 @@
+        --gpu_cluster_props=GPU_CLUSTER_PROPS
+            Type: Optional[str]
+            Default: None
+            Path of a file (json/yaml) containing configurations of the GPU cluster on which the Spark
+            applications is planned to be migrated.
+            The path is a local filesystem, or gstorage url.
+            This option does not require the cluster to be live.
+            When missing, the configurations are considered the same as the ones used by the 'cpu_cluster_props'.
+        --gpu_cluster_region=GPU_CLUSTER_REGION
+            Type: Optional[str]
+            Default: None
+            The region where the GPU cluster belongs to. Note that this parameter requires 'gpu_cluster_props' to be
+            defined.
+            When missing, the region is set to the value passed in the 'region' argument.
+        --gpu_cluster_zone=GPU_CLUSTER_ZONE
+            Type: Optional[str]
+            Default: None
+            The zone where the GPU cluster belongs to. Note that this parameter requires 'gpu_cluster_props' to be
+            defined.
+            When missing, the zone is set to the same zone as the 'cluster' on which the Qualification tool is submitted.
+        --debug=DEBUG
+            Type: bool
+            Default: False
+            True or False to enable verbosity to the wrapper script.
+        Additional flags are accepted.
+            A list of valid Qualification tool options. Note that the wrapper ignores the “output-directory“
+            flag, and it does not support multiple “spark-property“ arguments. For more details on
+            Qualification tool options, please visit https://nvidia.github.io/spark-rapids/docs/spark-qualification-tool.html#qualification-tool-options.
+    
+    ```
+
+- Example: Running Qualification tool on cluster that does not support Spark3.x
+    ```
+      spark_rapids_dataproc qualification \
+            --cluster dp-spark2 \
+            --region us-central1 \
+            --cpu_cluster_props=/tmp/test_cpu_cluster_prop.yaml \
+            --gpu_cluster_props=/tmp/test_gpu_cluster_prop_e2.yaml \
+            --eventlogs=gs://my-bucket/qualification_testing/dlrm_cpu/,gs://my-bucket/qualification_testing/tpcds_100in1/
+      2022-11-04 16:27:44,196 WARNING qualification: The cluster image 1.5.75-debian10 is not supported. To support the RAPIDS user tools, you will need to use an image that runs Spark3.x.
+      Failure Running Rapids Tool.
+              Tool cannot execute on the execution cluster.
+              Run Terminated with error.
+              The cluster image 1.5.75-debian10 is not supported. To support the RAPIDS user tools, you will need to use an image that runs Spark3.x.
+    ```
+
+- Example: Running Qualification tool passing list of google storage directories
+    - Note that the wrapper lists the applications with positive recommendations.
```

Comment:
Does "positive recommendations" mean positive speedup, savings, or both?

Reply:
Good catch @cindyyuanjiang !
this is the original README file we always had for the user-tools. I did not make changes to that file.
But you were right on spotting that and I fixed it to _positive savings_.